language/tools/move-mv-llvm-compiler/src/main.rs

```diff
@@ -281,14 +287,47 @@ fn main() -> anyhow::Result<()> {
                 break;
             }
         }
+        if entrypoint_generator.has_entries() {
+            let path = Path::new(&output_file_path);
+            entrypoint_generator.write_object_file(path.to_path_buf().parent().unwrap())?;
+        }
         // NB: context must outlive llvm module
         // fixme this should be handled with lifetimes
+        drop(entry_llmod);
         drop(global_cx);
     };
 
     Ok(())
 }
 
+fn initialize_logger() {
+    static LOGGER_INIT: std::sync::Once = std::sync::Once::new();
+    LOGGER_INIT.call_once(|| {
+        use env_logger::fmt::Color;
+        env_logger::Builder::from_default_env()
+            .format(|formatter, record| {
+                let level = record.level();
+                let mut style = formatter.style();
+                match record.level() {
+                    Level::Error => style.set_color(Color::Red),
```

Comment:
We could probably write tests checking the log messages. Is it worth the effort at the moment? I added this initializer for consistency with move-cli code path, as previously the logger was initialized when global context was created which is too late and wrong  (https://github.com/jcivlin/move/blame/599dbfbecc6d8b8893ee9c683b402e740f1f9820/language/tools/move-mv-llvm-compiler/src/stackless/translate.rs#L121-L142). I mentioned previously, I'd like to decommission move-mv-llvm-compiler as it will never be used by the end users, it is only used for testing at the moment, and it is taking wasted efforts to maintain.